src/dcurl.c

```diff
@@ -48,6 +51,11 @@ extern ImplContext PoWCL_Context;
 extern ImplContext PoWFPGAAccel_Context;
 #endif
 
+#if defined(ENABLE_REMOTE)
+extern RemoteImplContext Remote_ImplContext;
```

Comment:
I think we should change the name from `Remote_ImplContext` to `Remote_Context`.
Just like the naming of the previous delcared `ImplContext` structure variable.